lib/src/visitor/async_evaluate.dart

```diff
@@ -2292,7 +2292,7 @@ class _EvaluateVisitor
     var arguments = [
       for (var argument in node.arguments)
         await _visitCalculationValue(argument,
-            inMinMax: node.name == 'min' || node.name == 'max')
+            inMinMax: {'min', 'max', 'sqrt'}.contains(node.name))
```

Comment:
`inMinMax` shouldn't be true for `sqrt`, since that parameter is only necessary to provide the more lenient unit handling required for `min` and `max` to ensure compatibility with the existing global Sass functions.

This will probably be true for `round` and `abs` when you implement those, since they still need to support the legacy behavior, but in that case it would make sense to rename the parameter to something more general.